src/core/qgsfeaturerequest.cpp

```diff
@@ -56,7 +56,7 @@ QgsFeatureRequest::QgsFeatureRequest( const QgsExpression &expr, const QgsExpres
 {
 }
 
-QgsFeatureRequest::QgsFeatureRequest( const QgsFeatureRequest &rh )
+QgsFeatureRequest::QgsFeatureRequest( const QgsFeatureRequest &rh ): mTransformContext{}
```

Comment:
I'm wondering here if mTransformContext shouldn't rather be copied by the operator= ?

Reply:
Yes, you're right...

Reply:
done